### PR TITLE
Channel configuration on settings page

### PR DESCRIPTION
**Pull request by wuan**
Contains
  * model classes for channel
  * localization code generation
  * Android release preparation

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -25,8 +25,14 @@ apply plugin: 'com.android.application'
...
 
+def keystoreProperties = new Properties()
+def keystorePropertiesFile = rootProject.file('key.properties')
+if (keystorePropertiesFile.exists()) {
+    keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+}
+
 android {
-    compileSdkVersion 28
+    compileSdkVersion 30
```
Comment (scflode) [inline android/app/build.gradle:35]: Can we go lower than this? I’m not sure about the current distribution of SDK versions but we want to provide the app to as many people as possible ofc.

File: android/app/src/profile/AndroidManifest.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.mobile">
+    package="de.dealog.mobile.pilot">
```
Comment (scflode) [inline android/app/src/profile/AndroidManifest.xml:2]: Do we want to go with the `pilot` suffix?

**Review by scflode — APPROVED** [review]
LGTM. The comments are just for being sure we have the maximum possible compatibility and a good naming

File: android/app/build.gradle (hunk 1 above)
Comment (Rednox) [reply android/app/build.gradle:35]: https://flutter.dev/docs/resources/faq#what-devices-and-os-versions-does-flutter-run-on
Looks fine for me

Comment (wuan) [reply android/app/build.gradle:35]: That only determines which version of the SDK you have to use to build the application. `minSdkVersion 16` determines the minimum allowed version which matches the version supported by Flutter.

File: android/app/src/profile/AndroidManifest.xml (hunk 2 above)
Comment (wuan) [reply android/app/src/profile/AndroidManifest.xml:2]: Just added it to have a package for the pilot version, which we are working on for now. This allows us to have two versions in the play store, a released version for the "pilot" and a closed beta supporting the whole country.

Comment (wuan) [reply android/app/src/profile/AndroidManifest.xml:2]: Other possibility is to remove `pilot` and use a beta channel for the full release later. When we make that available, the pilot users will be updated to the full version.

Comment (scflode) [reply android/app/src/profile/AndroidManifest.xml:2]: Leave it for now. We can change this also later.

**Pull request merged**